benefits/core/migrations/0002_sample_data.py

```diff
@@ -160,15 +162,16 @@ def load_sample_data(app, *args, **kwargs):
         name="OAuth claims via Login.gov",
         eligibility_type=type1,
         auth_provider=auth_provider,
-        selection_label=_("eligibility.pages.index.oauth.label"),
+        selection_label=_("eligibility.pages.index.mst_login.label"),
         selection_label_description=None,
-        start_content_title=_("eligibility.pages.start.oauth.content_title"),
-        start_item_name=_("eligibility.pages.start.oauth.items[0].title"),
-        start_item_description=_("eligibility.pages.start.oauth.items[0].text"),
-        start_blurb=_("eligibility.pages.start.oauth.p[0]"),
-        unverified_title=_("eligibility.pages.unverified.oauth.title"),
-        unverified_content_title=_("eligibility.pages.unverified.oauth.content_title"),
-        unverified_blurb=_("eligibility.pages.unverified.oauth.p[0]"),
+        start_headline=_("eligibility.pages.start.mst_login.headline"),
+        start_sub_headline=_("eligibility.pages.start.mst_login.sub_headline"),
+        start_item_name=_("eligibility.pages.start.mst_login.items[0].title"),
+        start_item_description=_("eligibility.pages.start.mst_login.items[0].text"),
+        start_blurb=_("eligibility.pages.start.mst_login.p[0]"),
+        unverified_title=_("eligibility.pages.unverified.mst_login.title"),
+        unverified_content_title=_("eligibility.pages.unverified.mst_login.content_title"),
```

Comment:
Maybe we should rename this to `mst_login.headline` to align with the change on line 167 - and slowly start using headline/sub headline instead of content title.